packages/esm-home-app/src/homepage-header-date/homepage-header-date.component.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```

Comment:
This is more of a nitpicky concern than anything, but one of the suggestions in our [naming conventions](https://o3-docs.openmrs.org/docs/coding-conventions#naming) is to avoid [context duplication](https://github.com/kettanaito/naming-cheatsheet#avoid-context-duplication). We're in the home app, so that bit of context is already established. I think that naming the directory and its component with the `page-header` context is preferable. So the context reads as `home -> page-header` and then the child components could be broken down further into smaller pieces guided by the [design documentation](https://zeroheight.com/23a080e38/p/8358ed-page-header) and the functional requirements.

Reply:
Very insightful, will adhere.